DigitalLearningSolutions.Web/Styles/trackingSystem/sideNavMenus.scss

```diff
@@ -0,0 +1,49 @@
+﻿@import "~nhsuk-frontend/packages/core/all";
+
+.side-nav-menu {
+  @include govuk-media-query($until: desktop) {
+      display: none;
+  }
+}
+
+.side-nav__heading {
+  font-weight: 600;
+  color: #4c6272;
+  margin-bottom: 12px;

```

Comment:
We should also probably consider using the nhs defined spacing too rather than explicit px sizes